__tests__/components/Comments/WriteComment.test.js

```diff
@@ -31,16 +31,7 @@ describe('WriteComment should run correctly', () => {
   });
 
   it('styles should run correctly', () => {
-    expect(styles.replyToContainer(true)).toEqual({
-      marginLeft: 90,
-      fontFamily: fonts.inter[600],
-      marginBottom: 11,
-      marginTop: 7,
-      lineHeight: 15,
-      fontSize: 12,
-      color: COLORS.blackgrey
-    });
-    expect(styles.replyToContainer(false)).toEqual({
+    expect(styles.replyToContainer).toEqual({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code seems to have removed the conditionality of the `styles.replyToContainer` style. If this was intentional and all instances where this style is used have been updated accordingly, then it's fine. However, if there are still places in the code where the old conditional style is expected, this could cause issues.
```diff
-    expect(styles.replyToContainer).toEqual({
+    expect(styles.replyToContainer(true)).toEqual({
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->